### DISCUSSION TITLE
Hovering over a node

### DISCUSSION THREAD

**Discussion by 0xAnon0602** [bcakmakoglu/vue-flow#1355] [Q&A] [answered]
I want the website to display a tooltip type thing when I hover over a node displaying details of the node 
How can I achieve this ?

![image](https://github.com/bcakmakoglu/vue-flow/assets/71931823/0f7a8bc2-c54b-4f18-883c-9628d939ba3b)

**Comment by 0xAnon0602** [comment, marked as answer]
Nevermind I was able to do it with node-toolbar )

**Reply by lock4u** [reply]
Could u show me the code？
